From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 12 Aug 2013 15:22:26 +0200
Subject: texinfo strong note

Mollify makeinfo with regard to this issue:

$ makeinfo --output=mit-scheme-user.info user.texinfo
user.texinfo:1552: warning: @strong{Note...} produces a spurious cross-reference in Info; reword to avoid that.

The relevant portion of the manual states:

    Caution: Do not use `@strong' with the word `Note'; Info will
    mistake the combination for a cross reference.  (It's usually
    redundant, anyway.)  Use a phrase such as *Please notice* or
    *Caution* instead, or ...
---
 doc/user-manual/user.texinfo | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/user-manual/user.texinfo b/doc/user-manual/user.texinfo
index e154038..b5a14a4 100644
--- a/doc/user-manual/user.texinfo
+++ b/doc/user-manual/user.texinfo
@@ -1549,7 +1549,7 @@ storage available after collection, an exact non-negative integer.
 to system tasks after the need for a garbage collection is detected and
 before the garbage collector is started.  (An example of such a system
 task is changing the run-light to show ``gc'' when scheme is running
-under Emacs.)  @strong{Note well} that you should not specify
+under Emacs.)  @strong{Caution:} you should not specify
 @var{safety-margin} unless you know what you are doing.  If you specify
 a value that is too small, you can put Scheme in an unusable state.
 @end deffn
